src/components/Slot/Slot.tsx

```diff
@@ -43,7 +55,10 @@ class Slot extends UIComponent<Extendable<ISlotProps>, any> {
     as: 'div',
   }
 
-  renderComponent({ ElementType, classes, rest }) {
+  static create = createSlotFactory(Slot.defaultProps.as, content => ({ content }))
+  static createHTMLInput = createSlotFactory('input', type => ({ type }))
```

Comment:
Is there really need for creating this method? If we start creating this for the input, then we will continue creating one for all elements. It is pretty easily used generic.

Reply:
we're trying to keep consistency with the methods in `factories.tsx`; also, I think this is valuable because it's always this method we should use for creating an input same for the ones in `factories.tsx`

```tsx
// ============================================================
// HTML Factories
// ============================================================
export const createHTMLDivision = createShorthandFactory('div', val => ({ children: val }))
export const createHTMLIframe = createShorthandFactory('iframe', src => ({ src }))
export const createHTMLImage = createShorthandFactory('img', val => ({ src: val }))
export const createHTMLInput = createShorthandFactory('input', val => ({ type: val }))
export const createHTMLLabel = createShorthandFactory('label', val => ({ children: val }))
export const createHTMLParagraph = createShorthandFactory('p', val => ({ children: val }))
```

that will probably appear in `Slot.tsx` as:
```tsx
// ============================================================
// HTML Slot Factories
// ============================================================
static createHTMLDivision = createSlotFactory('div', children => ({ children }))
static createHTMLIframe = createSlotFactory('iframe', src => ({ src }))
static createHTMLImage = createSlotFactory('img', src => ({ src }))
static createHTMLInput = createSlotFactory('input', type => ({ type }))
static createHTMLLabel = createSlotFactory('label', children => ({ children }))
static createHTMLParagraph = createSlotFactory('p'', children => ({ children }))
```